clusterloader2/pkg/measurement/common/simple/etcd_metrics.go

```diff
@@ -104,15 +104,15 @@ func (e *etcdMetricsMeasurement) String() string {
 	return etcdMetricsMetricName
 }
 
-func (e *etcdMetricsMeasurement) startCollecting(provider, host string, interval time.Duration) {
+func (e *etcdMetricsMeasurement) startCollecting(provider, cmd string, host string, interval time.Duration) {
```

Comment:
will do.

Reply:
Should `func SSH(cmd, host, provider string)` get refactored as well?

Reply:
Maybe instead of `provider, host, cmd string`, let's make `cmd, host, provider string` in our etcd metric functions signatures.
I agree that we want coherent style. Good observation.